client/src/components/SidebarContainer.jsx

```diff
@@ -74,6 +76,26 @@ const SidebarContainer = (props: Props) => {
     fetchArrestData();
   }, [locationInfo, locationInfo.city, locationInfo.state, range]);
 
+  useEffect(() => {
+    async function fetchYearlyData() {
```

Comment:
I think it would be better to create an endpoint that retrieves arrest data from every year and aggregates it into a response.